patches/chrome-browser-ui-app_list-search-omnibox_provider.cc.patch

```diff
@@ -14,8 +14,8 @@ index 6e0aae5e3874fd3621175434e6335868996e798e..25eda5968661b4fb46af4f64ae43366a
                                   AppListControllerDelegate* list_controller)
      : profile_(profile),
        list_controller_(list_controller),
--      controller_(new AutocompleteController(
-+      controller_(new BraveAutocompleteController(
-           base::WrapUnique(new ChromeAutocompleteProviderClient(profile)),
+-      controller_(std::make_unique<AutocompleteController>(
```

Comment:
just marking these for potential chromium_src overrides

Reply:
yep